src/commands/install.rs

```diff
@@ -296,30 +296,34 @@ impl Target {
     }
 
     fn command(&self, url: &str, dev: bool) -> Command {
-        match self {
+        let shell_command = match self {
             Target::Npm => {
-                let mut cmd = Command::new("npm");
-                cmd.arg("install");
+                let mut cmd = format!("npm install {url}");
                 if dev {
-                    cmd.arg("--save-dev");
+                    cmd.push_str(" --dev");
                 }
-                cmd.arg(url);
                 cmd
             }
             Target::Yarn => {
-                let mut cmd = Command::new("yarn");
-                cmd.arg("add");
+                let mut cmd = format!("yarn add {url}");
                 if dev {
-                    cmd.arg("--dev");
+                    cmd.push_str(" --dev");
                 }
-                cmd.arg(url);
                 cmd
             }
             Target::Pip => {
-                let mut cmd = Command::new("pip");
-                cmd.arg("install").arg(url);
-                cmd
+                format!("pip install {url}")
             }
+        };
+
+        if cfg!(target_os = "windows") {
+            let mut cmd = Command::new("cmd");
+            cmd.arg("/C").arg(shell_command);
```

Comment:
I would actually leave it as is and just do `.arg(format!("{cmd:?}"))` instead of handling strings myself

Reply:
Can you show me a snippet of what you mean?

Reply:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=96ac77719eab17c711ced1d62298137b

Reply:
I think I see what you meant now. 

Are you suggesting passing the `cmd` variable that was originally constructed in our `match` to something like `Command::new("cmd.exe").arg("/C").arg(format!("{cmd:?}"))`? That way we reuse `Command`'s shell quoting logic.

So something like this:

```rust
let original_cmd = match self {
    Target::Npm => {
        let mut cmd = Command::new("npm");
        cmd.arg("install");
        if dev {
            cmd.arg("--save-dev");
        }
        cmd.arg(url);
        cmd
    }
    ...
};

if cfg!(target_os = "windows") {
    let mut cmd = Command::new("cmd");
    cmd.arg("/C").arg(format!("{original_cmd:?}"));
    cmd
} else {
    ...
}
```

Reply:
Yes that's what I meant, it also makes this PR shorter